apps/main/src/dex/components/PageDashboard/components/SummaryTotal.tsx

```diff
@@ -13,8 +13,9 @@ import {
 import { Chip } from '@ui/Typography'
 import Spinner from '@ui/Spinner'
 
-const SummaryTotal: React.FC = () => {
+const SummaryTotal = () => {
   const { dashboardDataMapper, isValidAddress } = useDashboardContext()
+  const { isLoading } = useDashboardContext()
```

Comment:
You're already using it above, could've been put on a single line?

Reply:
oh sorry I'll do that

Reply:
https://github.com/curvefi/curve-frontend/pull/696/commits/9ef9505f055c9a3d4268c11e9dc5dda56c9b4c5d